source/blog.liquid

```diff
@@ -0,0 +1,40 @@
+---
+layout: has_no_sidebars
+title: Sass Blog
+pagination:
+  data: collections.blog
+  size: 5
+  alias: posts
+  reverse: true
+permalink: '/blog/{% if pagination.pageNumber > 0 %}page/{{ pagination.pageNumber | plus: 1 }}/{% endif %}index.html'
+---
+
+{% assign pageNumber = pagination.pageNumber | plus: 1 %}
+{% assign pages = pagination.pages.size %}
+
+{% if pages > 1 %}
+  <p>Page {{ pageNumber }} of {{ pages }}</p>
+  {% if pagination.href.previous %}
+    <p>
+      <a href="{{ pagination.href.previous }}">Previous page</a>
+    </p>
+  {% endif %}
+{% endif %}
+
+<ul class="sl-c-list">
+  {% for post in posts %}
+    <li>
```

Comment:
I think the only difference in terms of markup from the old site to the new one is that we added a <ul> with <li>s instead of just a long list of <p> tags.